static/js/pages/SitesDashboard.tsx

```diff
@@ -98,29 +114,39 @@ export default function SitesDashboard(): JSX.Element {
             </Link>
           </div>
         </div>
-        <StudioList>
-          {listing.results.map((site: Website) => (
-            <StudioListItem
-              title={site.title}
-              subtitle={siteDescription(site) ?? ""}
-              to={siteDetailUrl.param({ name: site.name }).toString()}
-              key={site.uuid}
-            >
-              {site.publish_date ? (
-                <div className="text-success">Published</div>
-              ) : (
-                <div className="text-dark">Draft</div>
-              )}
-            </StudioListItem>
-          ))}
-        </StudioList>
+        {isPending || !listing ? (
+          <div className="site-page container">Loading...</div>
+        ) : (
```

Comment:
**Opinion / Minor Suggestion:** I think we not show the "Loading ...". These results should load fairly quickly, and if we don't show the loading, the previous results stay visible until the new results arrive, right? The very fast "loading" flash looks strange to me (opinion based!)

Reply:
I think you're right, the old results should stick around. and since I also changed it to show the surrounding initially it's less important to have a loading state I think